mixer2-localmail/README.md

```diff
@@ -0,0 +1,25 @@
+HTML mail sample with FakeSMTP. see
+https://nilhcem.github.io/FakeSMTP/
+
+Start FakeSMTP.
+```
+java -jar fakeSMTP.x.x.jar
+```
+Set port:8080, directory:any directory, and start server.
```

Comment:
This port number is not compatible to "well-known port".
80: http server
8080: web-ap server
25: mail server

So, this port number shuld be 25 or 1025 (as test purpose mail server)